docs/faq/pingfederate.md

```diff
@@ -0,0 +1,9 @@
+# Frequently Asked Questions - PingFederate
+
+## What versions of PingFederate are supported?
+
+The PingFederate provider supports versions `11.2` through `12.0` of PingFederate.
```

Comment:
maybe we link to this: https://registry.terraform.io/providers/pingidentity/pingfederate/latest/docs#pingfederate-version-support if we always intend to maintain that?